platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/layers/Layer.java

```diff
@@ -27,9 +28,17 @@ public void setProperties(@NonNull PropertyValue<?>... properties) {
     for (PropertyValue<?> property : properties) {
       Object converted = convertValue(property.value);
       if (property instanceof PaintPropertyValue) {
-        nativeSetPaintProperty(property.name, converted);
+        if(converted instanceof Expression) {
```

Comment:
Could this be part of `convertValue()`? (And space between `if` and `(`)

Reply:
wow, don't know how I could have missed that